dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -65,6 +66,24 @@ window.Dashboard = {
 		Marbles.history.navigate('login'+ redirectPath);
 	},
 
+	__isCertInstalled: function() {
+		var self = this, dispatchIfCertIsMissing = function(a, b) {
+			if (a + b !== 200 + 200) {
```

Comment:
If `https` ping fails is we're at `https://dashboard.CLUSTER_DOMAIN` (i.e. `window.location.protocol === 'https:'`), I think it's reasonable to assume the cert isn't installed and we can redirect to `http://dashboard.CLUSTER_DOMAIN`.